workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/types.d.ts

```diff
@@ -311,6 +311,7 @@ interface Product {
 	productId: number;
 	productStatus: number;
 	productType: string;
+	skus: sku[];
```

Comment:
Are you using the sku type right? because it is defined lower in capital letters a little further down in this same document, on line 360.